src/utils/Validate/validateRequired.js

```diff
@@ -16,6 +16,14 @@ const validateRequired = (value, label = '') => {
     }
   }
   if (!hasValue) {
+    if(customErrors && Array.isArray(customErrors)){
```

Comment:
You don't really need the first part of this as `Array.isArray(undefined|null)` will simply evaluate to `false`.
```suggestion
    if (Array.isArray(customErrors)) {
```